frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureTogglesHeader/CreateFeatureDialog.tsx

```diff
@@ -262,6 +272,34 @@ export const CreateFeatureDialog = ({
                                     />
                                 }
                             />
+                            <MultiSelectConfigButton
+                                tooltip={{
+                                    header: 'Select tags',
+                                }}
+                                description={configButtonData.tags.text}
+                                selectedOptions={tags}
+                                options={allTags.map((tag) => ({
+                                    label: `${tag.type}:${tag.value}`,
+                                    value: `${tag.type}:${tag.value}`,
+                                }))}
+                                onChange={setTags}
+                                button={{
+                                    label:
+                                        tags.size > 0
+                                            ? `${tags.size} selected`
+                                            : 'Tags',
+                                    labelWidth: `9ch`,
```

Comment:
What's the label width based on here? Maybe we could do something like:
```suggestion
                                    labelWidth: `${"nn selected".length}ch`,
```

to make it clear what it needs to fit? I also think it should be at least 10ch long, but probably 11 😅

Reply:
Makes sense. Updated